spec/property_sets_spec.rb

```diff
@@ -339,6 +339,21 @@
     end
   end
 
+  describe "update_attribute" do
```

Comment:
so this test would have failed before ?
... should use `update` and `update!` in the tests ?
...

Reply:
I'm glad you asked this because we must have deleted the better tests before committing... Updating this